docker/Makefile

```diff
@@ -15,9 +15,11 @@ staging-up:
 	PLATFORM=$(shell uname -m) \
 	ENVIROMENT=deployed/staging \
 	COMPOSE_DOCKER_CLI_BUILD=1 \
-	docker compose -f docker-compose.yml -f deployed/docker-compose.observability.yml $(HEALTHBOT_OVERRIDE) up --build --remove-orphans
+	docker compose -f docker-compose.yml -f deployed/docker-compose.observability.yml $(HEALTHBOT_OVERRIDE) up --build --remove-orphans --detach
```

Comment:
I added the `--detach` since we want `make <env>-up` to finish and not block forever, so the action can actually finish.